lib/apple/index.js

```diff
@@ -112,8 +116,8 @@ exports.verifyPayment = function (payment, cb) {
 		return cb(null, result);
 	}
 
-
-	verify(apiUrls.production, { json: jsonData }, function (error, resultString) {
+	var url = sandbox ? apiUrls.sandbox : apiUrls.production;
```

Comment:
@M4v3R what was the status code?

https://developer.apple.com/library/ios/releasenotes/General/ValidateAppStoreReceipt/Chapters/ValidateRemotely.html

Seems Apple is using the same error code.
Did you verify with a sandbox productId and productName?

I'm currently using a patch version of this package where we basically send a payment secret and it does validate. I can run it against this fork to verify.